python/ray/util/tracing.py

```diff
@@ -54,13 +55,13 @@ def inject_current_context() -> Dict[Any, Any]:
         propagators.inject(dict.__setitem__, context_dict)
         return context_dict
 
-    def extract(context_dict: Dict[Any, Any]) -> Context:
+    def extract(context_dict: Dict[Any, Any]):
```

Comment:
why did you remove the return type annotations here?

Reply:
maybe I'm missing some context here on this, is there a reason for it?

Reply:
This function needs to be successfully imported when ray is initialized, even when otel is not installed, so we can't put the type annotation here. @kathryn-zhou, can you check if we can get away with this:
```suggestion
    def extract(context_dict: Dict[Any, Any]) -> Context:
```

Reply:
yep, that change works @simon-mo . will add the type annotations in strings for the others as well

Reply:
yup, was just going to suggest that